### ISSUE TITLE
source map not found

### ISSUE THREAD

**Issue by adnan1naeem** [getsentry/sentry-react-native#1133]
Hi i am facing same issue .I don't know what am i missing ?? It is working fine in debugger mode but fails to load the source map when error occurs.Though sourcemaps have been loaded in artifacts but still facing this error.Please review my screenschots and provide me solution.
<img width="892" alt="Screenshot 2020-10-17 at 11 37 29 PM" src="https://user-images.githubusercontent.com/22443038/96350940-dc969f00-10d1-11eb-9089-cf9829acdb10.png">
<img width="800" alt="Screenshot 2020-10-17 at 11 37 44 PM" src="https://user-images.githubusercontent.com/22443038/96350942-df918f80-10d1-11eb-8b89-bf4228c4ff6d.png">
<img width="83" alt="Screenshot 2020-10-17 at 11 38 11 PM" src="https://user-images.githubusercontent.com/22443038/96350943-e0c2bc80-10d1-11eb-8c73-e817f6f31129.png">
<img width="1181" alt="Screenshot 2020-10-17 at 11 38 15 PM" src="https://user-images.githubusercontent.com/22443038/96350945-e0c2bc80-10d1-11eb-83b0-bfeb68bc93af.png">

_Originally posted by @adnan1naeem in https://github.com/getsentry/sentry-react-native/issues/249#issuecomment-711062783_

**Comment by jennmueng** [comment]
@adnan1naeem First, you're using the wrong value for `dist` in `init`, that's the name of the file not the distribution. Second, your source maps have no `dist` attached to them, how are you uploading your source maps, if you use the cli you will need to specify a `--dist` when you uploading your source maps. The list will be displayed under the filename like this (3.1.3 in this case) if they're uploaded correctly: 

<img width="1021" alt="Screen Shot 2020-10-19 at 1 09 52 PM" src="https://user-images.githubusercontent.com/30991498/96407790-63e82d80-120c-11eb-9d71-b40db51e95ee.png">

**Comment by adnan1naeem** [comment]
> @adnan1naeem First, you're using the wrong value for `dist` in `init`, that's the name of the file not the distribution. Second, your source maps have no `dist` attached to them, how are you uploading your source maps, if you use the cli you will need to specify a `--dist` when you uploading your source maps. The list will be displayed under the filename like this (3.1.3 in this case) if they're uploaded correctly:
> 
> <img alt="Screen Shot 2020-10-19 at 1 09 52 PM" width="1021" src="https://user-images.githubusercontent.com/30991498/96407790-63e82d80-120c-11eb-9d71-b40db51e95ee.png">

Please tell me from where can i get distribution  name and  this is my sentry cli command ```sentry-cli  releases    --dist  ./android/app/build/generated/assets/react/release/index.android.bundle  -p 'react-native'  --dist  --org 'fatimaindustry'  files pocRelease upload-sourcemaps ./android/app/build/generated/sourcemaps/react/release/index.android.bundle.map   ./android/app/build/generated/assets/react/release/index.android.bundle --rewrite --strip-prefix ./index.android.bundle```??

**Comment by jennmueng** [comment]
@adnan1naeem The distribution name is whatever string you would pass into `Sentry.init` with the `dist` property. Usually it's some version string. In your case, it would look something like this, with some `dist`, and add the directory of your project in place of `/path/to/project/root`.

```bash
sentry-cli releases \
    files "pocRelease" \
    upload-sourcemaps \
    --dist {dist} \
    --strip-prefix {/path/to/project/root} \
    --rewrite ./android/app/build/generated/assets/react/release/index.android.bundle ./android/app/build/generated/sourcemaps/react/release/index.android.bundle.map

**Comment by adnan1naeem** [comment]
I have tried all above solution but still no luck.can you please look on to my repo and tell me what am i missing for build ??
https://github.com/adnan1naeem/sentry/tree/feature/release

**Comment by jennmueng** [comment]
@adnan1naeem What error does the CLI throw, and how do the source maps show up on the sentry dashboard after using the correct cli commands that I provided? 

I can look at your repo in a bit but I don’t think that would be much help, it’s just running the commands I gave you.

Although I would need to note that GitHub Issues isn’t a support forum, this is just a support probables and nor an issue with our SDK. I will close this now.
